ref/README.md

```diff
@@ -38,10 +38,17 @@ the build step. A simple way to do that is with the `file` command as shown belo
 
 ```
 $ file data/inputs/*
-data/inputs/c_sw_12x24.nc: NetCDF Data Format data
-data/inputs/c_sw_24x24.nc: NetCDF Data Format data
-data/inputs/c_sw_48x24.nc: NetCDF Data Format data
-data/inputs/c_sw_48x48.nc: NetCDF Data Format data
+inputs/yppm_0.0.1.nc:  NetCDF Data Format data
```

Comment:
I thought these old files were gone.  I do not understand.  Why put them back?  In the README.md?

Reply:
They are not gone.  This fixes a cut-and-paste error that I made when updating the README file.  I had copied it from the c_sw, and so the filenames were incorrect.  These files are not gone, they are still there.  PR #11 proposes to remove some of those files, but it hasn't happened yet.  This change is consistent with `develop`.